recipe/meta.yaml

```
@@ -41,8 +39,6 @@ outputs:
         - cuda-nvml-dev ={{ cuda_compiler_version }}       # [(cuda_compiler_version or "None").startswith("12")]
         - libnuma
         - rdma-core  # [linux64]
```

Is it worth using this on Linux generally (as it is packaged on other architectures)?

```suggestion
        - rdma-core
```

---

UCX only supports linux, removing would allow it to attempt building for other OSs, no?

---

We only build for Linux

https://github.com/conda-forge/ucx-split-feedstock/blob/e4eb94dfcc133122070c8fdc69d9ee742fb1eeae/recipe/meta.yaml#L11

Removing it would just mean `linux_aarch64` and `linux_ppc64le` could also use it

---

You're right, applied the change https://github.com/conda-forge/ucx-split-feedstock/pull/161/commits/1241fa72b70a463668b0a31e74c22215093a676b .